src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -140,6 +141,13 @@ public abstract class Exercise extends BaseExercise implements LearningObject {
     @JsonIncludeProperties({ "id" })
     private Set<PlagiarismCase> plagiarismCases = new HashSet<>();
 
+    @Column(name = "continuous_plagiarism_control_enabled")
+    private boolean continuousPlagiarismControlEnabled = false;
+
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
this change is not good. It will lead to many additional database fetches all over the place in Artemis, even if this config is only needed in very specific places. Please avoid `OneToOne` here, even with fetch type Lazy (because Hibernate will still do it eagerly)

Reply:
instead you can add the OneToOne on the other side and make it a unidirectional relationship PlagiarismChecksConfig --> Exercise and only load it when it is actually needed